Library/Homebrew/global.rb

```diff
@@ -15,6 +15,7 @@
 require "active_support/core_ext/array/access"
 require "active_support/i18n"
 require "active_support/inflector/inflections"
+require "byebug"
```

Comment:
Can't include this here unless it's a vendored dependency (which is undesirable for non-developer dependencies). Will need to be an argument or environment variable to install this.